tools/spinel-cli/SNIFFER.md

```diff
@@ -0,0 +1,94 @@
+# Spinel Sniffer Reference
+
+Any Spinel NCP node can be made into a promiscuous packet sniffer, and this 
+tool both intializes a device into this mode and outputs a pcap stream that
+can be saved or piped directly into Wireshark.
+
+## System Requirements
+
+| OS     | Minimum Version  |
+|--------|------------------|
+| Ubuntu | 14.04 Trusty     |
+| Mac OS | 10.11 El Capitan |
+
+| Language | Minimum Version  |
+|----------|------------------|
+| Python   | 2.7.10           |
```

Comment:
Do you plan to support other languages?  Also, `spinel-cli.py` does not support Python 3.x.  FYI, `test/scripts/thread-cert` supports both Python 2.x and 3.x.